src/main/java/com/iota/iri/network/TransactionRequester.java

```diff
@@ -1,9 +1,7 @@
 package com.iota.iri.network;
 
 import java.security.SecureRandom;
```

Comment:
Hmm on my IDE I see that it is not being used.
Since you changed imports automatically I would think that it would have been removed automatically.

It is out of context of this PR, so I don't care too much